src/components/form/Form.js

```diff
@@ -261,8 +261,8 @@ export default class FormComponent extends Component {
           this.setContent(element, this.render());
           if (this.subForm) {
             this.subForm.attach(element);
-            if (this.subForm._form.display === 'wizard' && this.root._form.display === 'wizard') {
-              this.root.redraw();
+            if (this.subForm._form.display === 'wizard' && this.parent.parent?._form?.display === 'wizard') {
```

Comment:
this.parent.parent is super fragile and opinionated. What happens if this nested form is inside a panel that is inside a wizard?